SRC/NETWORK/hget.c

```diff
@@ -1500,14 +1618,29 @@ void ResolveServerName()
             Terminate(Buffer);
         }
     }
-    
+
     TcpConnectionParameters->remoteIP[0] = regs.Bytes.L;
     TcpConnectionParameters->remoteIP[1] = regs.Bytes.H;
     TcpConnectionParameters->remoteIP[2] = regs.Bytes.E;
     TcpConnectionParameters->remoteIP[3] = regs.Bytes.D;
-    
+
+	if (bHTTPS)
+	{
+		if (bCheckCertificate)
+			TcpConnectionParameters->flags = TcpConnectionParameters->flags | 8 ; //set flags properly for TLS certificate validation
+		if (bCheckHostName)
+			TcpConnectionParameters->hostName =  (int)domainName; //set host name buffer
+		else
+			TcpConnectionParameters->hostName =  0; //clear host name buffer
+	}
+	else
+	{
+		TcpConnectionParameters->flags = 0 ; //set flags properly for regular tcp
+		TcpConnectionParameters->hostName =  0; //clear host name buffer

```

Comment:
I think this comment, and the other ones above in `->hostName =`, are redundant. It's clear enough what the code does without them.

Reply:
Done :)